resources/backup-vaults-access-policies.go

```diff
@@ -117,3 +120,10 @@ func (b *BackupVaultAccessPolicy) Remove() error {
 func (b *BackupVaultAccessPolicy) String() string {
 	return b.backupVaultName
 }
+
+func (b *BackupVaultAccessPolicy) Filter() error {
+	if strings.HasPrefix(b.backupVaultName, "aws/efs/automatic-backup-vault") {
+		return fmt.Errorf("cannot delete EFS automatic backups backup policy")
```

Comment:
If we return an error here does it fail the nuke? If so should return nil

Reply:
ok made that nil, thanks